visualization/dashboard/src/MLIDashboard/Controls/GlobalExplanationTab/GlobalExplanationTab.tsx

```diff
@@ -166,13 +153,53 @@ export class GlobalExplanationTab extends React.PureComponent<IGlobalExplanation
                     jointDataset={this.props.jointDataset}
                     yAxisLabels={[localization.GlobalTab.aggregateFeatureImportance]}
                     sortArray={this.state.sortArray}
+                    chartType={this.state.chartType}
                     startingK={this.state.startingK}
                     unsortedX={this.props.metadata.featureNamesAbridged}
                     unsortedSeries={this.activeSeries}
-                    topK={this.props.globalBarSettings.topK}
+                    topK={this.state.topK}
                     onFeatureSelection={this.handleFeatureSelection}
                     selectedFeatureIndex={this.state.selectedFeatureIndex}
                 />
+                <IconButton
+                    className={classNames.chartEditorButton}
+                    onClick={this.toggleCalloutOpen} 
+                    iconProps={{iconName: "AreaChart"}}
+                    id={this._chartConfigId}/>
+                {(this.state.calloutVisible) && <Callout
+                    className={classNames.callout}
+                    gapSpace={0}
+                    target={"#" + this._chartConfigId}
+                    isBeakVisible={false}
+                    onDismiss={this.closeCallout}
+                    directionalHint={ DirectionalHint.bottomRightEdge}
+                    setInitialFocus={true}
+                >
+                    <Text variant="medium" className={classNames.boldText}>{localization.DatasetExplorer.chartType}</Text>
+                    <ChoiceGroup selectedKey={this.state.chartType} options={this.chartOptions} onChange={this.onChartTypeChange}/>
+                    <SpinButton
+                        className={classNames.topK}
+                        styles={{
```

Comment:
Should we move this to GlobalExplanationTab.styles.ts? 
This is something I've noticed in some places in fairLearn too and while i've also left it as it is but maybe adding a //ToDo for next pass will help?